### PR TITLE
MTP-2407 fremhevet innhold endringer

### PR DESCRIPTION
**Pull request by petterkoding**
added $transition-ms-... variables to use for smoother background-color transitions

### CODE REVIEW

File: mt-kit/core/css/src/global/_links.scss [hunk 1]
```diff
@@ -423,6 +426,10 @@ $transportlink-colors: 'error' !default;
       }
     }
 
+    &.b-r-radius {
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:429]: There is already a class that does this. `.box-border-radius-br-l`

**Review by lha047 — CHANGES_REQUESTED** [review]
You need to update all variations of fremhevet innhold. 
Should it have the big border radius? See comment

File: mt-kit/core/css/src/global/_links.scss (hunk 1 above)
Comment (petterkoding) [reply mt-kit/core/css/src/global/_links.scss:429]: will change and delete the class i created

**Review by lha047 — APPROVED** [review]
Good work :D

**Pull request merged**